scipy/io/matlab/_mio5_utils.pyx

```diff
@@ -2,15 +2,13 @@
 
 '''
 
-'''
-Programmer's notes
-------------------
-Routines here have been reasonably optimized.
+# Programmer's notes
+# ------------------
+# Routines here have been reasonably optimized.
 
-The char matrix reading is not very fast, but it's not usually a
-bottleneck. See comments in ``read_char`` for possible ways to go if you
-want to optimize.
-'''
+# The char matrix reading is not very fast, but it's not usually a
+# bottleneck. See comments in ``read_char`` for possible ways to go if you
+# want to optimize.
```

Comment:
if you write this as a single block, the linter won't complain, it'll recognise it as the docstring
```

''' Cython mio5 utility routines (-*- python -*- like)

Programmer's notes
------------------
Routines here have been reasonably optimized.
The char matrix reading is not very fast, but it's not usually a
bottleneck. See comments in ``read_char`` for possible ways to go if you
want to optimize.
'''
```

Reply:
Right; I think the idea was for the programmer's notes not to appear in the module-level docstring, but in this case it probably doesn't matter, given that it's internal.